tests/Tests/ORM/Functional/ProxiesLikeEntitiesTest.php

```diff
@@ -1,144 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Doctrine\Tests\ORM\Functional;
-
-use Doctrine\Tests\Models\CMS\CmsAddress;
-use Doctrine\Tests\Models\CMS\CmsArticle;
-use Doctrine\Tests\Models\CMS\CmsEmail;
-use Doctrine\Tests\Models\CMS\CmsGroup;
-use Doctrine\Tests\Models\CMS\CmsPhonenumber;
-use Doctrine\Tests\Models\CMS\CmsTag;
-use Doctrine\Tests\Models\CMS\CmsUser;
-use Doctrine\Tests\OrmFunctionalTestCase;
-use Doctrine\Tests\Proxies\__CG__\Doctrine\Tests\Models\CMS\CmsUser as CmsUserProxy;
-
-use function assert;
-
-/**
- * Test that Doctrine ORM correctly works with proxy instances exactly like with ordinary Entities
- *
- * The test considers two possible cases:
- *  a) __initialized__ = true and no identifier set in proxy
- *  b) __initialized__ = false and identifier set in proxy and in property
- *
- * @todo All other cases would cause lazy loading
- */
-class ProxiesLikeEntitiesTest extends OrmFunctionalTestCase
```

Comment:
IMO this test is to be kept: it's more about interactions of the proxy with the ORM, than the proxy itself

Reply:
OK, probably needs to be made fixed on 3.x then